### PR TITLE
Add scripts to create a job in CENTOS CI to test operator updates

### PR DESCRIPTION
**Pull request by flacatus**
This PR add scripts to check updates on minikube platform. This script have next tasks:

- Install minikube
- Install eclipse che previous stable version and then update to last stable version
- Create workspace using chectl 

Ref. issue: https://github.com/eclipse/che/issues/15814

### CODE REVIEW

**Comment by benoitf** [comment]
hello, as external point of view, it looks complex vs using https://github.com/marketplace/actions/minikube-single-node-kubernetes-action or kind to setup minikube

**Comment by flacatus** [comment]
@benoitf Thank you! For me using minikube for this job in CENTOS CI will be more comfortable. But I want to use this github action in chectl repo. There I have more problems with centos ci...

File: .ci/cico_updates_minikube.sh [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+# Catch_Finish is executed after finish script.
+catchFinish() {
+  result=$?
+  if [ "$result" != "0" ]; then
+    printInfo "Failed on running tests. Please check logs or contact QE team (e-mail:codereadyqe-workspaces-qe@redhat.com, Slack: #che-qe-internal, Eclipse mattermost: 'Eclipse Che QE'"
+    printInfo "Logs should be availabe on http://artifacts.ci.centos.org/devtools/che/che-eclipse-minikube-updates/${ghprbPullId}/"
+    exit 1
+    getCheClusterLogs
+    archiveArtifacts "che-eclipse-minikube-updates"
```
Comment (rhopp) [inline .ci/cico_updates_minikube.sh:26]: I think it would be nice to have repo name in the artifacts path. For example `che-operator-minikube-updates`

**Comment by tolusha** [comment]
One general remark, the workflow should be:
- deploy Eclipse Che
- create a workspace
- update Eclipse Che
- check if workspace can be started

**Comment by flacatus** [comment]
[test]

**Pull request merged**